src/drawing/DrawingList.tsx

```diff
@@ -0,0 +1,130 @@
+import React from 'react';
+import classNames from 'classnames';
+import cloneDeep from 'lodash/cloneDeep';
+import noop from 'lodash/noop';
+import DrawingTarget from './DrawingTarget';
+import useOutsideEvent from '../common/useOutsideEvent';
+import { AnnotationDrawing, PathGroup, Shape } from '../@types';
+import { checkValue } from '../utils/util';
+import { getShape } from './drawingUtil';
+
+export type Props = {
+    activeId?: string | null;
+    annotations: AnnotationDrawing[];
+    className?: string;
+    onSelect?: (annotationId: string | null) => void;
+};
+
+export const DRAWING_TARGET_PADDING = 10;
+
+export function filterDrawing({ target: { path_groups: pathGroups } }: AnnotationDrawing): boolean {
+    return pathGroups.reduce<boolean>(
+        (prevGroups, { paths }) =>
+            prevGroups &&
+            paths.reduce<boolean>(
+                (prevPaths, { points }) =>
+                    prevPaths &&
+                    points.reduce<boolean>(
+                        (prevPoints, { x, y }) => prevPoints && checkValue(x) && checkValue(y),
+                        true,
+                    ),
+                true,
+            ),
+        true,
+    );
+}
+
+export function scaleDrawing(
+    annotationDrawing: AnnotationDrawing,
+    { height, width }: { height: number; width: number },
+): AnnotationDrawing {
+    const scaledAnnotationDrawing = cloneDeep(annotationDrawing);
+    const {
+        target: { path_groups: pathGroups },
+    } = scaledAnnotationDrawing;
+
+    pathGroups.forEach(({ paths }) => {
+        paths.forEach(({ points }) => {
+            points.forEach(point => {
+                const { x, y } = point;
+                point.x = (x / 100) * width;
+                point.y = (y / 100) * height;
+            });
+        });
+    });
+
+    return scaledAnnotationDrawing;
+}
+
+export function sortDrawing({ target: targetA }: AnnotationDrawing, { target: targetB }: AnnotationDrawing): number {
+    const shapeA = getShape(targetA.path_groups);
+    const shapeB = getShape(targetB.path_groups);
+
+    return shapeA.height * shapeA.width > shapeB.height * shapeB.width ? -1 : 1;
+}
+
+export function getShapeWithPadding(
+    pathGroups: PathGroup[],
+    rootDimension: { height: number; width: number },
```

Comment:
Seems like this type is declared multiple times. I think a type alias for it already exists for regions. Can we reuse that?